src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -8906,11 +8906,16 @@ export interface OfficeAPI {
    *
    * @param divisionName - Name of the division
    * @param city - Name of the city
-   * @param job - Name of the job
+   * @param job - Name of the job. Do NOT use "Unassigned"; otherwise, this API does not do anything and just return false.
```

Comment:
The "otherwise" was kinda ambiguous.
```suggestion
   * @param job - Name of the job. Passing "Unassigned" will cause this API to not do anything and just return false.
```